specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/preview/2021-06-01-preview/deviceupdate.json

```diff
@@ -3356,11 +3378,9 @@
         },
         "deviceList": {
```

Comment:
You've changed this property from a list of device ids to a list of "DeviceUpdateAgentId"s -- shouldn't you change the property name to something like "deviceUpdateAgentsList"?

Reply:
We talked about that and we felt it wasn't necessary to change the property name. If needed, we could make the change for GA.